changelogs/fragments/7626-redfish-info-add-boot-progress-property.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - redfish_utils.py - adding BootProgress field to Systems inventory properties. This will add this data for hosts running Redfish 1.13+ per DMTF docs.
```

Comment:
Could you please adjust this so it tells a user what actually changes? The readers of this changelog are not collection developers, but end-users who use modules/plugins from the collection. So something like:
```
minor_changes:
  - redfish_info - ... (https://github.com/ansible-collections/community.general/pull/7626).
```
Thanks!

Reply:
Thanks for that! I've tweaked the description to highlight the module and where this change happens.